free-1.md

```diff
@@ -57,12 +108,27 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think that most of this project will be working on the **language** aspects.
+Once the back end methods of creating a fade function/color change function/etc..
+are completed, the rest will be trying to design the language in a way that promotes
+chaining commands and easily integrates aspects of looping and time to allow for complex
+light effects. 
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This project could be quite ambitious. There are lots of features that could be made for the user
+and it is easy to get lost in "feature creating land" instead of language design land.
+However, I also think the scope is not too large since it focuses on a reletively specific
+domain. 
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This could be a great idea for a project since it is very fun, allows for a lot of language design
+choices, and is mainly designed around allowing the user to create complex creations with it. 
+This might not be a good idea because it technically requires working with Arduino/Raspberry pi (even
```

Comment:
I'm a little confused with why this is a concern. The issue of working with Arudino and Raspberry Pi was not mentioned in the semantics section, so I'm not certain what concerns there are. For what it's worth, I've dabbled working with a Raspberry Pi, and it's often relatively quick to learn. I'm not certain about using it with respect to LEDs, but I'm certain there's online documentation and example projects!